### PR TITLE
Better instructions for Sublime Text

### PR DESCRIPTION
**Pull request by m-mitchell**
More explicit composer steps and fixed instructions for tab-width/encoding/extensions settings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,11 +132,7 @@ If you are using Sublime Text:
                 "@disable": false,
                 "args": [
                 ],
-                "encoding": "utf-8",
-                "excludes": [],
-                "extensions": "php",
-                "standard": "~/.composer/vendor/bin/silverorange/coding-standard/src/Silverorange/ruleset.xml",
-                "tabWidth":
+                "standard": "~/Code/custom-rulesets/phpcs.xml"
```
Comment (gauthierm) [inline README.md]: this needs to point to the phpcs.xml file above, right?

**Review by gauthierm — COMMENTED** [review]
Does this support different standards for different projects or does it enforce one standard across all projects?

**Comment by m-mitchell** [comment]
This will enforce one standard across all projects. As far as I can tell Sublime doesn't support adding extension config entries to a Sublime project config file, and using Sublime projects doesn't seem to work very well unless you're mounting the remote servers as a disk because the paths to the files keep changing.

**Comment by gauthierm** [comment]
@meg SublimeLinter-phpcs docs say you can use per-project configs https://github.com/SublimeLinter/SublimeLinter-phpcs#per-project-standards

**Comment by gauthierm** [comment]
For more background, the way we are going to use PHPCS is to commit a default rule file in each project. If the package-level composer-installed version of phpcs is used to do the linting it will use the correct rules automatically. Different projects may have significantly different rulesets (silverorange legacy vs silverorange modern).

**Comment by m-mitchell** [comment]
Oh, you're right! I tried setting it up this way and it still requires you to mount the remote as a disk (the sublime file needs to sit inside the actual project root, and sublime needs to have a path to it) but it works pretty well so long as you have PHPCS installed in the project. It overrides whatever you set up as a global default. Most of our projects aren't set up with PHPCS yet though. Is that something I should be doing as I go along? Some of our projects are only using our local composer server, which doesn't have PHPCS yet, and some of the public packages only know about packagist so they can't download all the silverorange packages that they require. E.g. I don't think it usually matters that composer fails to install requirements for silverorange/site, but here it would mean you can't run PHPCS on it?

**Comment by m-mitchell** [comment]
Also would we want to add those sublime project files to the repo?

**Comment by gauthierm** [comment]
> Some of our projects are only using our local composer server, which doesn't have PHPCS yet, and some of the public packages only know about packagist so they can't download all the silverorange packages that they require.

There are two plans to deal with this:

 1. release our public (open-source) packages on packagist in addition to our local composer repo
 2. update our composer configuration to allow authenticated external access

**Comment by gauthierm** [comment]
> Most of our projects aren't set up with PHPCS yet though. Is that something I should be doing as I go along?

Yes. Legacy packages should use a setup like the one in silverorange/swat. New ones should use the suggested modern silverorange standard.

**Comment by gauthierm** [comment]
> It overrides whatever you set up as a global default. 

That's exactly what we need. The default can be `SilverorangeLegacy` but it should also allow easily working on newer projects that are set to use `Silverorange` (aka PSR-2)

**Pull request merged**